cdap-ui/app/cdap/components/PipelineConfigurations/Store/ActionCreator.js

```diff
@@ -233,12 +234,15 @@ const updatePipeline = () => {
     }
   );
 
-  publishObservable.subscribe(() => {
-    PipelineDetailStore.dispatch({
-      type: PipelineDetailActions.SET_CONFIG,
-      payload: { config },
-    });
-  });
+  // Use pipe/map and not subscribe to not break error handling on other observers
+  publishObservable.pipe(
```

Comment:
if we do the following, would it still break other observers?

```
return pubishObservable.map((returnVal) => {
  PipelineDetailStore.dispatch({
    type: PipelineDetailActions.SET_CONFIG,
    payload: { config },
  });

  return returnVal;
});
```

Reply:
Good point. Let me check.